recipes/modelcards/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "modelcards" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/modelcards-{{ version }}.tar.gz
```

Comment:
Do not ship[ a global tests/ module](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=550518&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2814). This is a bug that should be fixed upstream because it is also happening in the PYPI release.

[This line](https://github.com/nateraw/modelcards/blob/2e11195f607d4ccfc696991fde4f1275817ac851/setup.py#L27) needs to exclude the tests.

Reply:
Done